### PR TITLE
Improving Tutorial

### PR DESCRIPTION
**Pull request by tylerweitzman**

### CODE REVIEW

File: P8-New-Note-Controller/content.md [hunk 1]
```diff
@@ -69,9 +62,9 @@ This will create a new `Navigation Controller`, sure it looks a bit ugly however
...
 Great, now how do we connect the `Add` button to the `New Note View Controller` ?  
 
-Seque to the rescue!
+Segue to the rescue!
 
-##Seques
+##Segues
 
 A segue is a smooth transition. (Pronounced SEG-way, to avoid that awkward interview moment)
 Seques allow you to create transitions from one scene to another easily, you will be happy to know they are nice and easy to use.
```
Comment (extrajordanary) [inline P8-New-Note-Controller/content.md]: Please find and change all segue typos in this file

File: P7-Local-Storage-Realm/content.md [hunk 2]
```diff
@@ -186,6 +186,20 @@ Remember when you added the `UITableViewDataSource` protocol extension? These fu
     return Int(notes?.count ?? 0)
 >
 
+In the above code, note how the operator "??" handles the case in which the variable notes is nil and empty.
```
Comment (extrajordanary) [inline P7-Local-Storage-Realm/content.md]: be sure to style vars etc as code

File: P11-Search/content.md [hunk 3]
```diff
@@ -140,6 +140,6 @@ Once again a good time to **Commit your code.**
 Well done! You have made it this far and have a fully functional Notes application.  
 Sure, it may not be super pretty and polished yet. However, it's your first App and a great starting place in your development.
 
-The next chapter is brief chapter on app polish. We will look at changing the color of various elements to start putting your own stamp on it.
+The next chapter is brief chapter on app polishing. We will look at changing the color of various elements to start putting your own stamp on it.
```
Comment (extrajordanary) [inline P11-Search/content.md]: is a brief

File: P10-Keyboard-Input/content.md [hunk 4]
```diff
@@ -144,15 +144,15 @@ The **MakeSchool ConvenienceKit** helps us out here by wrapping things up a litt
     keyboardNotificationHandler = KeyboardNotificationHandler()
 >        
         keyboardNotificationHandler!.keyboardWillBeHiddenHandler = { (height: CGFloat) in
-            UIView.animateWithDuration(0.3, animations: { () -> Void in
+            UIView.animateWithDuration(0.3) {
```
Comment (extrajordanary) [inline P10-Keyboard-Input/content.md]: code PR

File: P8-New-Note-Controller/content.md (hunk 1 above)
Comment (cocojoe) [inline P8-New-Note-Controller/content.md]: this does make me laugh now, not sure why I keep typing as seque

**Comment by cocojoe** [comment]
I like the ?? explanation, please pull from upstream and tidy up commit so it can be auto merged.  thx

File: P8-New-Note-Controller/content.md [hunk 5]
```diff
@@ -83,8 +76,13 @@ Let's try one out right now and connect our '+' button to the `New Note View Con
 >
 > ![image](add_create_segue_1.png)
 
+<<<<<<< HEAD
```
Comment (samaratrilling) [inline P8-New-Note-Controller/content.md]: Leftover from merging?

File: P8-New-Note-Controller/content.md [hunk 6]
```diff
@@ -83,8 +76,13 @@ Let's try one out right now and connect our '+' button to the `New Note View Con
...
+You will be presented with an additional dialog of segue types, for now we are going to use *Show*.  This will push the `New Note View Controller` to the top of
+The Navigation stack.
+=======
 You will be presented with an additional dialog of segue types: for now we are going to use *Show*.  This will push the `New Note View Controller` to the top of
 the Navigation stack.
+>>>>>>> upstream/master
```
Comment (samaratrilling) [inline P8-New-Note-Controller/content.md]: Same here

File: P8-New-Note-Controller/content.md [hunk 7]
```diff
@@ -83,8 +76,13 @@ Let's try one out right now and connect our '+' button to the `New Note View Con
 >
 > ![image](add_create_segue_1.png)
 
+<<<<<<< HEAD
+You will be presented with an additional dialog of segue types, for now we are going to use *Show*.  This will push the `New Note View Controller` to the top of
+The Navigation stack.
+=======
```
Comment (samaratrilling) [inline P8-New-Note-Controller/content.md]: And here

**Pull request merged**